packages/ses/src/whitelist.js

```diff
@@ -546,6 +547,7 @@ export const whitelist = {
     constructor: '%SharedError%',
     message: 'string',
     name: 'string',
+    cause: 'undefined',
```

Comment:
```suggestion
    cause: false,
```

Apparently it was a [bug](https://github.com/tc39/notes/blob/master/meetings/2021-10/oct-26.md#:~:text=However%2C%20Chrome%2093,and%20node%2016.11.) in v8's implementation to have included it in the first place:
> However, Chrome 93 and node 16.9 and 16.10 shipped with `cause` property on `Error.prototype` as well, which is not supposed to be there. It's technically allowed by the spec, but that was not the design of the feature or the intention and it's not what the other engines have chosen to do. And so, of course, that obvious bug was fixed in Chrome 94 and node 16.11.